.gitignore

```diff
@@ -18,3 +18,4 @@ man/*
 *.idea
 .tox/*
 *.bak
+Pipfile*
```

Comment:
FYI, there's been discussion in #743 of adding Pipfile to the git repository as a tracked file.